app/components/ui/Navigation/NavigationFocusReset.tsx

```diff
@@ -1,16 +1,24 @@
 import React, { useEffect, useRef } from "react";
 import { useLocation } from "react-router-dom";
 
-// Resets focus state for a11y, by focusing and unfocusing an off-screen element at the top of the page when page location changes
-
-export const NavigationFocusReset = () => {
+/* 
+  Resets focus state for a11y, by focusing and unfocusing an off-screen element at the top of the page when page location changes
+  Since filters change the location, the logic is to only reset focus when a main-level page change occurs (not when user filters a search)
```

Comment:
Realizing this would be strange behavior if we were to traverse from say /services/123 to /services/124. Rethinking.

Reply:
Handled in c320d26